app/src/renderer/vuex/getters.js

```diff
@@ -1,15 +1,13 @@
 import BN from "bignumber.js"
 import { calculateTokens } from "scripts/common"
 // ui
-export const config = state => state.config
-
 export const filters = state => state.filters
 export const notifications = state => state.notifications
-export const user = state => state.user
+export const session = state => state.session
 export const lastPage = state => {
```

Comment:
do we need / use `lastPage` since we removed the back button?

Reply:
still used in "sign in > redirect" process